From aba357c350faf0600a68e6848dac4e907af15eb6 Mon Sep 17 00:00:00 2001
From: Pierre Tardy <pierre.tardy@intel.com>
Date: Wed, 9 Jan 2013 16:47:25 +0100
Subject: [PORT FROM MAIN] [PATCH] fix call of my-dir outside of the beginning of Android.mk

BZ: 79013

Change-Id: I823ccd5afbc0e46d3d88d45a28018f7e8706f872
Orig-Change-Id: I035aa0c011258aecca39e4f453acbd2ba944e7ac
Signed-off-by: Pierre Tardy <pierre.tardy@intel.com>
Reviewed-on: http://android.intel.com:8080/85889
Reviewed-by: Alberdi, Ion <ion.alberdi@intel.com>
Reviewed-by: Monnier, Olivier <olivier.monnier@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 Android.mk |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 8faadaa..9b8ab12 100644
--- a/Android.mk
+++ b/Android.mk
@@ -31,10 +31,10 @@ include $(MY_WEBRTC_ROOT_PATH)/src/modules/audio_coding/codecs/isac/fix/source/A
 include $(MY_WEBRTC_ROOT_PATH)/src/common_audio/signal_processing/Android.mk
 
 # build .so
-LOCAL_PATH := $(call my-dir)
+LOCAL_PATH := $(MY_WEBRTC_ROOT_PATH)
 
 include $(CLEAR_VARS)
-include $(LOCAL_PATH)/../../external/webrtc/android-webrtc.mk
+include $(LOCAL_PATH)/android-webrtc.mk
 
 LOCAL_ARM_MODE := arm
 LOCAL_MODULE := libwebrtc_audio_preprocessing
-- 
1.7.4.1

